sale_outplacement/models/project.py

```diff
@@ -42,37 +42,49 @@ class ProjectTask(models.Model):
     @api.model
     def init_joint_planning(self, outplacement_id):
```

Comment:
Many similar lines of code, I'll have an alternative implementation I'll discuss with you